pulumi/src/saltstack/master/cloud-init.go

```diff
@@ -75,7 +75,7 @@ fileserver_backend:
 
 gitfs_remotes:
   - https://github.com/tinkerbell/infrastructure:
-      - root: saltstack/states
+      - root: saltstack/salt
```

Comment:
Is this needed?

Reply:
It is to go along with the rename of the directory in the repository to make the testkitechen stuff happy.